### PR TITLE
docs: improve functions documentation

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

**Review by mariux — APPROVED** [review]
amazing.. ;)

File: docs/functions/tm_plantimestamp.md
Comment (mariux) [inline docs/functions/tm_plantimestamp.md:1]: i do not think we have this information.

**Review by mariux — CHANGES_REQUESTED** [review]

File: docs/functions/tm_plantimestamp.md
Comment (i4ki) [reply docs/functions/tm_plantimestamp.md:1]: we do not! I will remove it, thanks!

Comment (i4ki) [reply docs/functions/tm_plantimestamp.md:1]: removed

**Review by MordecaiMalignatus — APPROVED** [review]

**Comment by i4ki** [comment]
@mariux can you review again?

**Comment by i4ki** [comment]
@soerenmartius can you have a look?

**Pull request merged**